cves/CVE-2015-5144.yml

```diff
@@ -48,8 +48,18 @@ description_instructions: |
   keep too.
 
   Your target audience is people just like you before you took any course in
-  security
-description:
+  security.
+description: |
+  The vulnerability was in Django's built in validators such as the EmailValidator,
+  URLValidator, the validate_ip4_address (validator for ip4 addresses), and the
+  validate_slug validator (which validates the part of a url that is used when
+  accessing a resource).
+
```

Comment:
A short description of what Django validators are could be helpful for people that are unfamiliar with Django.

Reply:
Hi @JetMas , I just added a short description of what Django validators are. Could you confirm that this is enough clarification for people unfamiliar with Django? Thanks!

Reply:
Yes, I think the description is enough.